src/contract/deploy.rs

```diff
@@ -46,17 +48,26 @@ impl<T: Transport> Builder<T> {
     pub fn execute<P, V>(self, code: V, params: P, from: Address) -> Result<PendingContract<T>, ethabi::Error>
     where
         P: Tokenize,
-        V: Into<Vec<u8>>,
+        V: AsRef<str>,
     {
         let options = self.options;
         let eth = self.eth;
         let abi = self.abi;
 
+        let mut code_hex = code.as_ref().to_string();
```

Comment:
we probably should get rid of leading `0x` (if there is any)

Reply:
This is done in the end.
https://github.com/tomusdrw/rust-web3/pull/209/files#diff-0dfcbf3acd5476076104fa446de24e33R63